reference-architectures/deploy-arch-ibm-slz-vpc-standard.md

```diff
@@ -0,0 +1,125 @@
+---
+
+copyright:
+  years: 2023
+lastupdated: "2023-03-28"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: "Vincent Burckhardt"
+
+# The release that the reference architecture describes
+version: 1.0
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: url #TODO
+
+docs: https://cloud.ibm.com/docs/secure-infrastructure-vpc
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/reference-architectures/vpc.drawio.svg
+
+related_links:
+  - title: "VPC landing zone - Standard variation"
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-vsi-ra"
+    description: "A deployable architecture that is based on the IBM Cloud for Financial Services reference and that provides virtual servers in a secure VPC for your workloads."
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+compliance: FedRAMP
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+<!--
+Don't include "reference architecture" in the following title.
+Specify a title based on a use case. If the architecture has a module
+or tile in the IBM Cloud catalog, match the title to the catalog. See
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# VPC landing zone - Standard variation
+{: #vsi-ra-qs}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+
+The Standard variation of the VPC landing zone deployable architecture uses two Virtual Private Clouds (VPC), a Management VPC, and a Workload VPC to manage the environment and the deployed workload. Each VPC is a multi-zoned, multi-subnet implementation that keeps your workloads secure.
+
+Virtual Private Endpoints connect the VPCs to each other and to IBM Cloud services. Cloud services include Key Protect, IAM, Container Registry, and Cloud Object Storage. The IBM Cloud services are configured for storage and key management.
```

Comment:
Transit gateway is what connects the VPC's together and Virtual private endpoints are to IBM Cloud services

Reply:
I would take this out and leave it open for interpretation - 
"Cloud services include Key Protect, IAM, Container Registry, and Cloud Object Storage. The IBM Cloud services are configured for storage and key management."

Reply:
Updated in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/41